detections/cloud/aws_detect_users_creating_keys_with_encrypt_policy_without_mfa.yml

```diff
@@ -0,0 +1,22 @@
+author: "Rod Soto, Splunk"
+date: "2020-10-27"
+description: "This search provides detection of accounts creating KMS with encrypt policy and without using MFA."
+how_to_implement: "You must install splunk AWS add on and Splunk App for AWS. This search works with clodtrail logs"
+id: c79c164f-4b21-4847-98f9-cf6a9f49179e
+known_false_positives: "Not all KMS key creations are malicious. Particulary look for kms:Encrypt only policy. It is very unusual to create a key to encrypt only and not to decrypt."
+name: "aws detect users creating keys with encrypt policy without mfa"
```

Comment:
@rosplk : Can you update all the search names to use upper case for the first letter of every word?

Reply:
@patel-bhavin is there a SPL style guide (like PEP 8 for python) or reference we can use when checking best practices for a particular searches readability?

Reply:
yeah, there is an old one which I compiled but it needs a refresher:  https://confluence.splunk.com/display/MG/ESCU+Best+Practices

Reply:
also, after doing some testing: looks like the search syntax and logic is in correct.